ivy/functional/ivy/activations.py

```diff
@@ -95,8 +108,8 @@ def leaky_relu(
     ret
         The input array with leaky relu applied element-wise.
 
-    Functional Examples
-    -------------------
+    Examples
+    --------
 
```

Comment:
ivy/functional/ivy/activations.py:94 in public function `leaky_relu`:
        D412: No blank lines allowed between a section header and its content ('Examples')

Reply:
Hello @Daniel4078 , as far as I can say, leaky_relu is a different function tracked by someone alse on the specific issue: https://github.com/unifyai/ivy/issues/6626

As I understand I should not interfere with someone else work, is that correct? In any case, if you still want me to fix this error it is pretty easy and minor and of course I'll handle it. Can you please clarify?

Reply:
Well, I just saw in the change log and find this error is from a place that you changed. Of course you can fix existing errors by others as long as no new critical error are introduced. Thank you!